File: PapagoTransOne.Start.py
# -*- coding: utf-8 -*- 

# 모니터 해상도 : 1920 * 1080
import time
import sys
import codecs
import requests
#pip install pyperclip	클립
import pyperclip

#pip install pyautogui	키보드
import pyautogui  

from tkinter import Tk

PyMode = sys.argv[1]


if PyMode == "stop" :
	pyautogui.leftClick(x=438, y=340)
	time.sleep(2)
	pyautogui.hotkey('alt', 'F4')
	exit()
else :

	#번역기록삭제
	pyautogui.leftClick(x=1851, y=204)
	time.sleep(1)
	pyautogui.hotkey('enter')	#엔터
	time.sleep(1)




	pyautogui.leftClick(x=438, y=340)
	time.sleep(1)
	pyautogui.hotkey('ctrl', 'a')


	#코드 가져오기
	f = open("C:/xampp/htdocs/_Ntos/_Trans/_TransOne_codelist.txt", "r", encoding="utf8")
	itemcode = f.read()
	f.close()

	#내용 가져오기
	f = open("C:/xampp/htdocs/_Ntos/_Trans/_TransOne_namelist.txt", "r", encoding="utf8")
	itemname = f.read()
	f.close()
	time.sleep(2)
	pyperclip.copy(itemname)
	pyautogui.hotkey('ctrl', 'v')
	time.sleep(2)


	pyautogui.hotkey('tab')
	time.sleep(1)
	pyautogui.hotkey('ctrl', 'a')
	time.sleep(1)
	pyautogui.hotkey('ctrl', 'c')


	result = Tk().selection_get(selection="CLIPBOARD")

	time.sleep(1)
	URL = 'http://amazon.ntos.co.kr/_Mini_/_WinTrans/ItemWinTrans.php' 
	data = {'CustId': 'amazon', 'mode': 'up', 'TransType':'papago', 'codelist':itemcode, 'namelist' : result } 
	response = requests.post(URL, data=data)

File: PapagoTransOne.Start.php
<?php
include_once(dirname(__FILE__).'/_common.php');

$TransUrl = CF_getDataURL('http://amazon.ntos.co.kr/_Mini_/_WinTrans/ItemWinTrans.php?CustId=amazon&limit='.$limit);

$dir = 'C:/xampp/htdocs/_Ntos/_Trans';

//재등록
$sName = "_TransOne";

$PyMode = 'not';
switch($TransUrl){
	case 'stop' :
		$PyMode = "stop";
		passthru("/xampp/htdocs/_Ntos/_Trans/PapagoTransOne.Start.py $PyMode");



		$Chk_Hi = (int)date("Hi");
		$Chk_i = (int)date("i");
			
		$RandTime = 80;	//초

		$SetStart = mktime(date('H'),date('i'),date('s')+$RandTime,date('m'),date('d'),date('Y'));
		$SetStartDay = date('Y/m/d', $SetStart);

			exec('schtasks /delete /tn "'.$sName.'" /f');
			$SetStartTime = date('H:i', $SetStart);
			exec('schtasks /create /tn "'.$sName.'" /tr c:\xampp\htdocs\_Ntos\_Trans\PapagoTransOne.Ready.bat /sc once /st '.$SetStartTime.' /sd '.$SetStartDay);
			//			exec('schtasks /create /tn "'.$sName.'" /tr c:\xampp\htdocs\_Ntos\_Trans\PapagoTransOne.Ready.bat /sc minute /mo 3 /st '.$SetStartTime.' /sd '.$SetStartDay);

		exit;
	break;

	default :

		$pattern = '@<ItemList>(?P<ItemList>.*)</ItemList>@Us';
		preg_match_all($pattern, $TransUrl, $matches);
		$ItemList = $matches['ItemList'];

		$CodeTxt = array();
		$NameTxt = array();
		foreach($ItemList as $Key => $Val){
			$pattern = '@<code>(?P<Code>.*)</code>@Us';
			preg_match($pattern, $Val, $matches);
			$Code = $matches['Code'];
			$CodeTxt[] = $Code;

			$pattern = '@<name>(?P<Name>.*)</name>@Us';
			preg_match($pattern, $Val, $matches);
			$Name = $matches['Name'];
			$NameTxt[] = $Name;

		}

		$CodeTxt = implode("\n\n", $CodeTxt);
		$dest_file = $dir."/_TransOne_codelist.txt";
		$source_file = fopen($dest_file, "w") or die("Unable to open file!");
		fwrite($source_file, $CodeTxt);
		fclose($source_file);


		$NameTxt = implode("\n\n", $NameTxt);
		$dest_file = $dir."/_TransOne_namelist.txt";
		$source_file = fopen($dest_file, "w") or die("Unable to open file!");
		fwrite($source_file, $NameTxt);
		fclose($source_file);

		Sleep_fun();
		passthru("/xampp/htdocs/_Ntos/_Trans/PapagoTransOne.Start.py  $PyMode");

	break;
}	//end switch


if($TransUrl != "stop") {

}	//end if
//파일삭제
unlink($dir."/_TransOne_codelist.txt");
unlink($dir."/_TransOne_namelist.txt");



$Chk_Hi = (int)date("Hi");
$Chk_i = (int)date("i");
	
$RandTime = 60;	//초

$SetStart = mktime(date('H'),date('i'),date('s')+$RandTime,date('m'),date('d'),date('Y'));
$SetStartDay = date('Y/m/d', $SetStart);



if(empty($limit)){
	if($Chk_Hi >= "900" && $Chk_Hi <= "925"){
		$SetStartTime = "09:29";

		exec('taskkill /im chrome.exe /f');
		exec('schtasks /delete /tn "'.$sName.'" /f');
		exec('schtasks /create /tn "'.$sName.'" /tr c:\xampp\htdocs\_Ntos\_Trans\PapagoTransOne.Ready.bat /sc once /st '.$SetStartTime.' /sd '.$SetStartDay);

	} else if($Chk_Hi >= "1010" && $Chk_Hi <= "1030"){
		$SetStartTime = "10:35";

		exec('taskkill /im chrome.exe /f');
		exec('schtasks /delete /tn "'.$sName.'" /f');
		exec('schtasks /create /tn "'.$sName.'" /tr c:\xampp\htdocs\_Ntos\_Trans\PapagoTransOne.Ready.bat /sc once /st '.$SetStartTime.' /sd '.$SetStartDay);

	} else {
		exec('c:\xampp/php/php.exe -f "c:\xampp\htdocs\_Ntos\_Trans/PapagoTransOne.Start.php"');
	}
} else {
		exec('c:\xampp\php\php.exe -f "c:\xampp\htdocs\_Ntos\_Trans\PapagoTransOne.Start.php"');
}	//end if $limit